cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -0,0 +1,140 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform Wikipedia Pipeline Application
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _examples-wikipedia-data-pipeline:
+
+==================
+Wikipedia Pipeline
+==================
+
+A Cask Data Application Platform (CDAP) example demonstrating a typical data processing pipeline using CDAP Workflows.
+
+Overview
+========
+
+This example demonstrates a CDAP application performing analysis on Wikipedia data using MapReduce and Spark programs
+running within a CDAP Workflow: *WikipediaPipelineWorkflow*.
+
+This example can be run in both online and offline modes.
+- In the online mode, the custom action *DownloadWikiDataAction* reads the *pageTitleStream* is one output element
+of the `Facebook "likes" API <https://developers.facebook.com/docs/graph-api/reference/v2.4/object/likes>`__.
+For each event, it tries to download Wikipedia data for the page using the
+`MediaWiki Wikipedia API <https://www.mediawiki.org/wiki/API:Main_page>`__. It stores the downloaded data in the
+KeyValueTable dataset *wikiData*.
+- In the offline mode, it expects Wikipedia data formatted per the output of the MediaWiki API in the stream *wikiStream*.
+
+The MapReduce program *wikiDataToDataset* consumes this stream and stores it in the same KeyValueTable dataset
+ *wikiData*. Data can be uploaded to the *wikiStream* using the CDAP CLI.

```

Comment:
How does this render on the website? This space is actually between the words "dataset" and "*wikiData*". Anyway, I've moved the space to the previous line.